lib/config.js

```diff
@@ -228,6 +228,13 @@ config.chunksMaxParallelUpload = 20;
  */
 config.isEnterprise = false;
 
+/**
+ * Used to sign JWTs that PrairieLearn provides to PrairieTest for authentication.
+ * PrairieTest should be configured with the same value for
+ * `prairieLearnAuthSecret`.
+ */
+config.prairieTestAuthSecret = 'THIS_IS_A_SECRET_KEY';
```

Comment:
It may be more secure to make this `null` by default and only enable this feature if it's non null, that way we can't accidentally deploy both PL and PT with the same default secret.

Reply:
Or give them different default names on the PL/PT sides so that it is broken by default? Maybe `THIS_SHOULD_MATCH_THE_PT_KEY` (and correspondingly `..._PL_...`).

Reply:
I like this a lot! I'll change that.

Reply:
Done in 190c788.